tests/internal/remoteconfig/test_remoteconfig.py

```diff
@@ -124,7 +131,7 @@ def test_remote_config_forksafe():
 
         if os.fork() == 0:
             assert RemoteConfig._worker is not None
-            assert RemoteConfig._worker is not parent_worker
+            assert RemoteConfig._worker is parent_worker
```

Comment:
Does this mean that the new intended behavior is for both parent and child processes to share the same `RemoteConfig._worker`?